src/HtmlProcessor/CssToAttributeConverter.php

```diff
@@ -94,10 +91,6 @@ private function mapCssToHtmlAttributes(array $styles, \DOMElement $node): void
      * Tries to apply the CSS style to $node as an attribute.
```

Comment:
As this change is adding backticks for parameter names in the main description elsewhere, I suggest we change this too:

```suggestion
     * Tries to apply the CSS style to `$node` as an attribute.
```